sql_generators/glean_usage/templates/baseline_clients_daily_v1.query.sql

```diff
@@ -24,15 +24,18 @@ WITH base AS (
     normalized_channel,
     normalized_os,
     normalized_os_version,
-    {% if has_distribution_id %}
-    metrics.string.metrics_distribution_id AS distribution_id,
-    {% else %}
-    CAST(NULL AS STRING) AS distribution_id,
-    {% endif %}
     {% if app_name == "fenix" %}
+    metrics.string.metrics_distribution_id AS distribution_id,
     metrics.string.first_session_install_source AS install_source,
+    CAST(NULL AS BOOLEAN) AS is_default_browser,
+    {% elif app_name == "firefox_desktop" %}
+    metrics.string.usage_distribution_id AS distribution_id,
```

Comment:
out of curiosity, how did you determine that we need the `usage_` prefix here for desktop? i couldn't determine that it would be there from the patch and it hasn't landed in the glean dictionary yet... is there a schema somewhere that you checked?

Reply:
I just checked the `firefox_desktop_stable.baseline_v1` table in BQ. And I just assumed this is the correct field here.